dashboard/components/BannerActions.vue

```diff
@@ -72,12 +75,15 @@ const WPDE_GITHUB_REPO = 'https://github.com/wmde/wikipedia.de-banners/blob/mast
 
 let editLink: string = props.isWPDE ? WPDE_GITHUB_REPO : CENTRAL_NOTICE_EDIT_URL.replace( '{{banner}}', bannerPageName.value );
 const isCompiled: boolean = !!props.compileInfo;
-
+let isCompiling: boolean = false;
+let isCopying: boolean = false;
```

Comment:
I think for this to work when you change those values in the methods later you have to use Refs here, like this:
```suggestion
let isCompiling: Ref<boolean> = ref( false );
let isCopying: Ref<boolean> = ref( false );
```
in the methods you can then change the values with
`isCompiling.value = true`;

Correct me if I was testing this wrong, but this code is only working for me when using refs currently. Else, the template values do not get updated and stay `false` (because of how the new "script setup" works)